policyengine_us/parameters/gov/states/wv/tax/income/subtractions/low_income_earned_income_exclusion/income_threshold.yaml

```diff
@@ -0,0 +1,26 @@
+description: West Virginia qualifies filers for the  low-income earned income exclusion with income below this threshold.
+metadata:
+  period: year
+  unit: USD
+  label: West Virginia low-income earned income exclusion federal adjusted gross income threshold
+  reference:
+  - title: Code of West Virginia §11-21-10 (b)
+    href: https://code.wvlegislature.gov/11-21-10/
+  - title: 2021 West Virginia Personal Income Tax Forms And Instructions-2021 FAMILY TAX CREDIT TABLES
+    href: https://tax.wv.gov/Documents/TaxForms/2021/it140.pdf#page=20
+  - title: 2022 West Virginia Personal Income Tax Forms And Instructions-2022 FAMILY TAX CREDIT TABLES
+    href: https://tax.wv.gov/Documents/PIT/2022/PersonalIncomeTaxFormsAndInstructions.2022.pdf#page=27
+
+  breakdown:
+    - filing_status
+
+SINGLE:
+  2022-01-01: 10_000
```

Comment:
was this introduced in 2022? 
If no - need 2021 value 

We also need to include in the subtractions.ymal file